spec/services/catalogue_services_client_spec.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe CatalogueServicesClient do
+  subject(:service) { described_class.new }
+
+  describe "#get_holdings" do
+    before do
+      cat_response = IO.read("spec/files/catalogue_services/08aed703-3648-54d0-80ef-fddb3c635731.json")
+      WebMock.stub_request(:get, /catservices.test\/catalogue-services\/folio\/instance\/08aed703-3648-54d0-80ef-fddb3c635731/)
+        .to_return(status: 200, body: cat_response, headers: {"Content-Type" => "application/json"})
```

Comment:
@CEdwardsBlasikiewicz the reason why you weren't getting a proper response back was because you weren't specifying a Content-Type header. Faraday will only parse responses with "application/json" headers into JSON automatically.